admin_guide/advanced_scheduling/pod_affinity.adoc

```diff
@@ -0,0 +1,150 @@
+[[admin-guide-sched-pod-affinity]]
+= Using Pod Affinity and Anti-Affinity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+_Pod affinity_ and _pod anti-affinity_ allows you to specify rules about how pods should be placed relative to one another. For example, you could spread or pack pods within a service or relative to pods in other services; or, you can make sure a pod is scheduled on a machine with an SSD on it. 
```

Comment:
`you can make sure a pod is scheduled on a machine with an SSD on it.` does not seem to fit for pod affinities or anti-affinities.